include/picongpu/fields/incidentField/profiles/DispersiveLaser.def

```diff
@@ -0,0 +1,104 @@
+/* Copyright 2013-2022 Axel Huebl, Heiko Burau, Anton Helm, Rene Widera,
+ *                     Richard Pausch, Alexander Debus, Sergei Bastrakov,
+ *                     Fabia Dietrich
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/fields/incidentField/profiles/BaseParam.def"
+
+namespace picongpu
+{
+    namespace fields
+    {
+        namespace incidentField
+        {
+            namespace profiles
+            {
+                namespace defaults
+                {
+                    struct DispersiveLaserParam : public BaseParam
+                    {
+                        /** Beam waist: distance from the axis where the pulse intensity (E^2)
+                         *              decreases to its 1/e^2-th part,
+                         *              at the focus position of the laser
+                         * W0_SI = FWHM_of_Intensity / sqrt{ 2* ln(2) }
+                         *                             [   1.17741    ]
+                         *
+                         * unit: meter
+                         */
+                        static constexpr float_64 W0_SI = 5.0e-6 / 1.17741;
+
+                        /** The laser pulse will be initialized PULSE_INIT times of the PULSE_LENGTH
+                         * WATCH OUT! Dispersion parameters may lead to pulse lenghtening
+                         * Please ensure to choose a value high enough to cover the whole pulse,
+                         * otherwise the Fourier Transformation will misbehave.
+                         *
+                         *  unit: none
+                         */
+                        static constexpr float_64 PULSE_INIT = 20.0;
+                        
+                        // Dispersion Parameters
+                        
+                        /** SD: spatial dispersion in focus
+                         * = dx0 / dΩ
+                         *
+                         * unit: m * s
+                         */
+                        static constexpr float_64 SD_SI = 0.0;
+                        
+                        /** AD: angular dispersion in focus
+                         * dθ/dΩ, e.g. tan(α_tilt)/Ω0
```

Comment:
I just skimmed over the code, but I noticed the greek letters like Ω,θ and α everywhere in the comments. I don't think we want to use unicode characters in PIConGPU. So `* d theta / d Omega, e.g. tan(alpha_tilt)/(Omega_0)` would probably better here (and in the other places with the greek letters).

Reply:
I hope I catched all.